packages/legend-query/src/stores/QueryBuilderState.ts

```diff
@@ -243,6 +251,32 @@ export class QueryBuilderState {
     } catch (error) {
       assertErrorThrown(error);
       this.changeClass(undefined, true);
+      const lambdaJson =
+        this.graphManagerState.graphManager.serializeRawValueSpecification(
```

Comment:
```suggestion
      const valueSpec =
        this.graphManagerState.graphManager.buildValueSpecification(
          this.graphManagerState.graphManager.serializeRawValueSpecification(
            new RawLambda(rawLambda.parameters, []),
          ),
          this.graphManagerState.graph,
        );
      const compiledValueSpecification = guaranteeType(
        valueSpec,
        LambdaFunctionInstanceValue,
      );
      const compiledLambda = guaranteeNonNullable(
        compiledValueSpecification.values[0],
      );
      processQueryParameters(compiledLambda, this);


```

Reply:
I tried using this. The issue with this is it calls `V1_buildLambdaBody` which should have something in `body` to process otherwise it throws an assertion error

Reply:
yeah changing

Reply:
Try the above